documentation/MeetingMinutes/2024/MOM-2024-01-31.md

```diff
@@ -0,0 +1,169 @@
+- Elizabeth points out that the Client Credentials flow and purpose usage should be reviewed. Jesús points out that this was discussed in an issue opened by Vodafone (issue #92), so after discussion this was the conclusion: Client Credentials can be used only when no personal user data is processed, so purpose does not apply. Elizabeth points out the purpose might be charging relevant. Jesús points out that it's a great point, but since there is an agreement, it should go to v0.3. Elizabeth agrees.
+- Axel asks for clarification on whether scope is not needed or purpose is not needed. Clarified that scope is always required. Axel asks if charging models can be differentiated by different scopes and Elizabeth points out that it's not possible. Candidate to be discussed in v0.3.
+
+<br>
+
+Regarding 1) "Format of the profile":
+- Axel points out that the main argument against a self-contained format is that it's very difficult to see what changes the standard. Points out that if we have a full document, we should clearly identify each change that changes/updates/deviates the standard.
+- Jesús points out that Telefónica first has tried to create a profile like Mobile Connect (which is self-contained), as requested in recent calls. On the other hand, Telefónica thinks that if we add references to standards and then the standard is not completely clear on a topic, different topics may have different interpretations. We have had this problem in the past. In any case, we agree that it does not make sense to copy everything from another reused standard, but only the relevant parts.
+- Axel will be fine with a note at the beginning explaining why we do it one way or the other. For example, explain that "if we repeat redirect_url and it's not clearly marked as different, a reader will have to go to auth2 to check if the text is the same or has changed". Indicates that if we choose to repeat, we should clearly indicate what has changed. Indicates that FAPI goes a different way, just with certain restrictions.
+- Shilpa indicates that she has no other opinion. Probably pros and cons of each option. Elizabeth would vote for a self-contained with a list of changes (change history), otherwise it wouldn't be possible to follow. Axel points out that if you're a developer reading this, you won't notice what's changed from the existing standard. Elizabeth points out that with a delta it's going to be impossible, she has 20 years experience in development.... Axel has been working in standards for 10 years, and mistakes happen, he thinks that if you rewrite something, you might invalidate the other standard.
```

Comment:
Shilpa indicates that I do not want to add a new option (in addition to the options that have been already proposed) but would suggest that we should ask for organization specific viewpoints (considering the pros and cons) and see if we can get a consensus on the approach.

Reply:
Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/118/commits/ffc2571d079773b87d1f37f1a389f846982d5f8b